sdk/servicebus/azure-servicebus/api_review.py

```diff
@@ -0,0 +1,586 @@
+# Two-Clients Approach
+
+## APIs:
+class SeviceBusSenderClient:
+    def __init__(
+        self,
+        fully_qualified_namespace : str,
+        entity_name : str,
+        credential : TokenCredential,
+        logging_enable: bool = False,
+        http_proxy: dict = None,
+        transport_type: TransportType = None,
+        retry_total : int = 3,
+    ) -> None:
+    @classmethod
+    def from_queue(
+        cls,
+        fully_qualified_namepsace : str,
+        queue_name : str,
+        credential : TokenCredential,
+        **kwargs
+    ) -> SeviceBusSenderClient:
+    @classmethod
+    def from_topic(
+        cls,
+        fully_qualified_namespace : str,
+        topic_name : str,
+        credential : TokenCredential,
+        **kwargs
+    ) -> SeviceBusSenderClient:
+    @classmethod
+    def from_connection_string(
+        cls,
+        conn_str : str,
+        entity_name : str = None,
+        **kwargs
+    ) -> SeviceBusSenderClient:
+
+    def __enter__(self):
+    def __exit__(self):
+
+    def close(self) -> None:
+
+    def get_properties(self) -> Dict[str, Any]:
```

Comment:
We call get_eventhub_properties for event hubs. Should this be consistent?

Reply:
`get_queue_properties`/`get_topic_properties`/`get_subscription_properties` looks more suitable here as those are the real entity that a client connects to.

However, we're using the unified sender/receiver so let's keep status quo first.

Reply:
Assuming that this is the http based api, this is out of scope for Preview 1